app/jobs/scheduled/rebake_gif_images.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Jobs
+  class RebakeGifImages < ::Jobs::Scheduled
+    every 1.hour
+
+    MAX_PROCESSED_GIF_IMAGES ||= 200
+
+    def execute(args)
+      Upload
+        .where("original_filename LIKE '%.gif'")
```

Comment:
Should we use `.where("extension = 'gif' || original_filename like '%.gif' ")` ... gifs can sneak in with the wrong filename we rely on actual content I think.

Reply:
It can be null though in some cases

Reply:
That sounds like a good idea. :+1: